src/api/entities/DividendDistribution/index.ts

```diff
@@ -273,6 +274,72 @@ export class DividendDistribution extends CorporateAction {
     return paidStatuses.map((paid, index) => ({ ...participants[index], paid }));
   }
 
+  /**
+   * Retrieve an Identity that is entitled to dividends in this Distribution (participant),
+   *   the amount it is entitled to and whether it have been paid or not
+   *
+   * @note if the Distribution Checkpoint hasn't been created yet, the result will be null.
+   *   This is because the Distribution participant cannot be determined without a Checkpoint
+   */
+  public async getParticipant(args: {
+    identity: string | Identity;
```

Comment:
make args optional and add `@param args.identity - defaults to the current Identity` to the docs